hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -359,7 +370,12 @@ def _test_mock_panda_container(use_gpu: bool, mock_container: BaseDeepSMILEPanda
     assert_test_step(module, data_module, use_gpu)
 
 
-@pytest.mark.skipif(no_gpu or True, reason="Test requires GPU | hot-fix: pathmnist link is down")
+def test_mock_tiles_panda_container_cpu(mock_panda_tiles_root_dir: Path) -> None:
```

Comment:
On a GPU machine is this the same as `test_mock_container_gpu parameterised` with `mock_panda_tiles_root_dir`? In which case should we skip it if GPU is available?

Reply:
the test is not marked gpu so it won't be picked to be run on GPU?

Reply:
Good point!